src/nl/surf/eduhub_rio_mapper/cli.clj

```diff
@@ -178,6 +180,31 @@
       (assoc (if (= type "opleidingsrelatiesBijOpleidingseenheid") ::rio/opleidingscode ::ooapi/id) id
              ::rio/type type)))
 
+(defn- make-async-callback [req]
```

Comment:
We seem to use the "make-" prefix for functions which build/construct some value or function.  Maybe prefix it with "do-" instead.